pandas/io/tests/json/test_pandas.py

```diff
@@ -959,6 +959,25 @@ def test_read_jsonl(self):
         expected = DataFrame([[1, 2], [1, 2]], columns=['a', 'b'])
         assert_frame_equal(result, expected)
 
+        # GH15132: ascii input of non-ascii unicode characters for Python 2
+        # \xe2\x80\x9d == \u201d == RIGHT DOUBLE QUOTATION MARK
+        if compat.PY2:
+            # simulate file handle
+            json = StringIO('{"a": "foo\xe2\x80\x9d", "b": "bar"}\n'
+                            '{"a": "foo", "b": "bar"}\n')
+            result = read_json(json, lines=True)
+            expected = DataFrame([[u"foo\u201d", "bar"], ["foo", "bar"]],
+                                 columns=['a', 'b'])
+            assert_frame_equal(result, expected)
+
+            # simulate string
+            json = ('{"a": "foo\xe2\x80\x9d", "b": "bar"}\n'
```

Comment:
Thanks for the feedback, @jreback. 

To make this work for PY3, is it OK to include ``# -*- coding: utf-8 -*-`` at the top of the ``pandas/io/tests/json/test_pandas.py`` file? If so, I can simply use the unicode character directly in the ``json`` string.

```python
# -*- coding: utf-8 -*-
...
json = '{"a": "foo”", "b": "bar"}\n{"a": "foo", "b": "bar"}\n'  # note the ” character
```

If not, I would suggest using a bytes string that is utf-8 decoded if PY3:

```python
json = b'{"a": "foo\xe2\x80\x9d", "b": "bar"}\n{"a": "foo", "b": "bar"}\n'
if compat.PY3:
     # PY3 will not interpret the ascii-equivalent \xe2\x80\x9d correctly without decoding.
    json = json.decode('utf-8')
```